recipes/genrich/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.5" %}
+{% set sha256 = "88491354ad1149847ef80bfedf06478087142ac3d0f7b72ebd9779c455afb87a" %}
+
+
+package:
+  name: genrich
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/jsh58/Genrich/archive/v0.5.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch
+
+about:
+  home:
+    https://github.com/jsh58/Genrich
+  license: MIT
```

Comment:
Please add `license_file: LICENSE` so the license is packaged in the distributed tarball